lib/pbench/test/unit/server/query_apis/test_controllers_list.py

```diff
@@ -25,11 +25,27 @@ def build_index(self, server_config, dates):
             index += f"{idx}{d},"
         return index
 
-    def test_missing_json_object(self, client, server_config):
+    def test_non_accessible_user_data(self, client, server_config, pbench_token):
+        """
+        test_missing_json_object Test behavior when Authorization header does not have access to other user's data
```

Comment:
I'm trying to stop repeating the function name in the docstring. It wastes space, and it can easily get out of date during refactoring. It's silly that the template even has it.